src/noAttributeDecoratorRule.ts

```diff
@@ -0,0 +1,61 @@
+import { IRuleMetadata, RuleFailure, WalkContext } from 'tslint';
+import { AbstractRule } from 'tslint/lib/rules';
+import {
+  ConstructorDeclaration,
+  Decorator,
+  forEachChild,
+  isConstructorDeclaration,
+  Node,
+  ParameterDeclaration,
+  SourceFile
+} from 'typescript';
+import { getDecoratorName } from './util/utils';
+
+export class Rule extends AbstractRule {
+  static readonly metadata: IRuleMetadata = {
+    description: 'Disallows usage of @Attribute decorator.',
+    options: null,
+    optionsDescription: 'Not configurable.',
+    rationale: '@Attribute is considered bad practice. Use @Input instead.',
+    ruleName: 'no-attribute-decorator',
+    type: 'functionality',
+    typescriptOnly: true
+  };
+
+  static readonly FAILURE_STRING = '@Attribute is considered bad practice. Use @Input instead.';
```

Comment:
Not sure about this message...

Reply:
If there's any place in the Angular Style Guide that says it's a bad practice to use `@Attribute` we should link it here, otherwise better rephrase this (and also the `rationale`).

Reply:
LGTM